src/SFA.DAS.EmployerAccounts/DependencyResolution/EncodingRegistry.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.Encoding;
+using StructureMap;
+
+namespace SFA.DAS.EmployerAccounts.DependencyResolution
+{
+    public class EncodingRegistry : Registry
+    {
+        public EncodingRegistry()
+        {
+            For<IEncodingService>().Use<EncodingService>().Singleton();
```

Comment:
We may not want to mix usage of the EncodingService and HashingService

Reply:
This one is for you@VasanthaKasirajan3008

Reply:
We need to use Encoding Service to consume Commitments Api.